embed-builder/src/builder.rs

```diff
@@ -165,16 +165,14 @@ pub enum EmbedErrorType {
     },
     /// The total content of the embed is too large.
     ///
-    /// Refer to [`EmbedBuilder::EMBED_LENGTH_LIMIT`] for more information about
-    /// what goes into this limit.
+    /// See [`EmbedBuilder::EMBED_LENGTH_LIMIT`].
```

Comment:
a lot of these don't directly relate to `Discord Docs` links. I'm sorry i wasn't clear enough but could you please revert these changes